Assets/js/17.pending_events.js

```diff
@@ -0,0 +1,68 @@
+Mautic.contactclientPendingEventsTable = function () {
+
+    if ("undefined" === mQuery.fn.dataTable) {
+        mQuery.getScript(mauticBasePath + '/' + mauticAssetPrefix + 'plugins/MauticContactLedgerBundle/Assets/js/datatables.min.js', function () {
```

Comment:
Don't like the dependency of a second plugin here.

Reply:
Did we ever move the datatables stuff when we did the global datatables thing? Need to go find that. Maybe its somewhere more core-ish?